lib/measured/case_sensitive_measurable.rb

```diff
@@ -1,7 +0,0 @@
-class Measured::CaseSensitiveMeasurable < Measured::Measurable
```

Comment:
You create a case-sensitive measurable through `case_sensitive true` in the `Measured.build` DSL now